src/Bicep.Core/CoreResources.Designer.cs

```diff
@@ -10,48 +10,35 @@
 
 namespace Bicep.Core {
     using System;
+    using System.Reflection;
```

Comment:
The Mac version of VS for some reason doesn't add comments when generating the .deisgner.cs file for a resx.